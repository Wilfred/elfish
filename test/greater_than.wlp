(defun main ()
  (print "(> 1 2): ")
  (print (if (> 1 2) "true\n" "false\n"))

  (print "(> 2 2): ")
  (print (if (> 2 2) "true\n" "false\n"))

  (print "(> 3 2): ")
  (print (if (> 3 2) "true\n" "false\n")))
