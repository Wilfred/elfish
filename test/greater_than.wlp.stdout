(> 1 2): false
(> 2 2): false
(> 3 2): true
